web/index.html

```diff
@@ -81,6 +76,13 @@
             caret-color: #ffffff;
         }
     </style>
+    <!--
+        Note: interactive-widget=resizes-content is ignored on iOS Safari. Working around that has proven to be
+        problematic and causing bugs. At the moment of writing, we don't really need to work around this. If we attempt
+        to do that again, we should do that carefully and minding the potential side effects.
+
+        Some context can be found in https://github.com/Expensify/App/issues/16082
+    -->
     <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1, interactive-widget=resizes-content">
```

Comment:
Do we even need `interactive-widget=resizes-content` here?

Reply:
I think it's respected on iOS Chrome, if I recall correctly

Reply:
The `interactive-widget` property doesn't affect auto-scroll. This affects viewport behavior when the virtual keyboard is shown.

https://developer.chrome.com/blog/viewport-resize-behavior/

`resizes-content` is what we want. It's supported mainly (or only?) by Chrome, but [it's present in CSS Viewport Module Level 1 Editor’s Draft](https://drafts.csswg.org/css-viewport/#interactive-widget-section), so one day maybe even iOS Safari will support it.

So it's unrelated and there's no need to change it.

Reply:
Can you please remove the comment then, I don't think we need explain anything here, it may be just confusing

Reply:
We're removing a hack that was a workaround for the fact that Safari doesn't support that property. So that's why I left a comment. But I'll remove it if you think it's confusing

Reply:
Hmm, I dunno, I think the comment is fine. This is a file we don't touch much, so I don't mind leaving it there.

Reply:
@s77rt Done